models/swimlanes.js

```diff
@@ -174,6 +174,21 @@ Swimlanes.helpers({
   },
 
   lists() {
+    const enabled = Meteor.user().hasShowDesktopDragHandles();
```

Comment:
hasShowDesktopDragHandles is separate feature, it should not affect any sorting, that should be separate option at lists menu